qiita_db/handlers/tests/test_artifact.py

```diff
@@ -34,6 +34,19 @@ def test_get_artifact(self):
 
 
 class ArtifactHandlerTests(OauthTestingBase):
+    def setUp(self):
+        super(ArtifactHandlerTests, self).setUp()
```

Comment:
Yeah, we haven't check about py3, it's down on the list ATM. Thanks for pointing it out! Maybe once we finish the more pressing things we can start thinking on doing the movement to py3 only...